source/Octopus.Tentacle.Tests.Integration/ClientFileTransferRetriesTimeout.cs

```diff
@@ -153,5 +148,14 @@ public class ClientFileTransferRetriesTimeout : IntegrationTest
             // Ensure we actually waited and retried until the timeout policy kicked in
             duration.Elapsed.Should().BeGreaterOrEqualTo(TimeSpan.FromSeconds(14));
         }
+
+        private class BooleanValues : IEnumerable
```

Comment:
Nit: If we are keeping this locally, maybe we should name this how we use it. E.g. `StopPortForwarderAfterFirstCallValues`?